lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_japan.govspeak.erb

```diff
@@ -55,6 +55,6 @@
              locals: { calculator: calculator } %>
 
 
-  You can pay by credit card, but not by personal cheque. The embassy in Tokyo will also accept cash.
+  <%= render partial: 'pay_by_credit_card_but_not_personal_cheque.govspeak.erb' %>
```

Comment:
Why did you decide not to include `tokyo` in the partial name? Given that the content includes Tokyo-specific text, this seems a bit dangerous.